sheet_to_triples/trans.py

```diff
@@ -80,9 +80,16 @@ def __repr__(self):
         return f'<{self.__class__.__name__} {self.name!r}>'
 
     @classmethod
-    def iter_from_name(cls, name):
-        transforms_dir = os.getenv('TRANSFORMS_DIR', 'transforms')
-        path = os.path.join(transforms_dir, name + '.py')
+    def iter_from_name(cls, name, path_behaviour='relative'):
```

Comment:
Strawman alternative interface:
```python
...(cls, name, base_path=None):
    if base_path is None:
        base_path = os.getenv('TRANSFORMS_DIR', 'transforms')
    ...
```

Reply:
Yeah that's probably better. Also shorter.